content/en/docs/concepts/services-networking/ingress-controllers.md

```diff
@@ -63,6 +63,9 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
 * [Voyager](https://appscode.com/products/voyager) is an ingress controller for
   [HAProxy](https://www.haproxy.org/#desc).
 * [Wallarm Ingress Controller](https://www.wallarm.com/solutions/waf-for-kubernetes) is an Ingress Controller that provides WAAP (WAF) and API Security capabilities.
+* [FortiADC Ingress Controller](https://docs.fortinet.com/document/fortiadc/7.0.0/fortiadc-ingress-controller-1-0/742835/fortiadc-ingress-controller-overview)
+  supports the Kubernetes Ingress resources and allows you to manage FortiADC objects from Kubernetes.
```

Comment:
The new link and the suggested changes all look good.

Reply:
Hi. We keep this list in alphabetical order. Please re-edit to follow the existing sort order.